app/scripts/lib/multichain-api/scope/authorization.ts

```diff
@@ -22,11 +21,6 @@ export type Caip25Authorization =
 export const processScopes = (
```

Comment:
Should this just be called `validateAndFlattenScopes` now?

Reply:
https://github.com/MetaMask/metamask-extension/pull/25873/commits/153667a5d4060bfbc555b384e5a7bf14975e3510